Source/WebGPU/WGSL/AST/ASTBinaryExpression.h

```diff
@@ -31,6 +31,7 @@ namespace WGSL::AST {
 
 enum class BinaryOperation : uint8_t {
     Add,
+    Times,
```

Comment:
Is it really named Times in the spec? otherwise I'd just use Multiply or smtg similar, though Myles might have a different opinion

Reply:
The spec doesn't name it AFAICT, but since it's the token and not the operation I was between `Times` and `Asterisk`, but I saw times in other places. I don't have a strong opinion, but it's also used for pointer dereference, so neither `Times` nor `Multiply` are very accurate 🤷‍♂️

Reply:
On a related note, we could also move to the model that just uses the character as the token "name", that would avoid having to name tokens, which is probably the hardest part of making these patches.

Reply:
Oh, I just realized I also named the operation `Times` 🤦 I was thinking this was the token. I'll rename before landing

Reply:
I actually like the "unify all binary operators into a single class, and give that class a `char m_operator;` model. The different binary operators are not structurally different.